frontend/appflowy_flutter/lib/plugins/database/board/presentation/board_page.dart

```diff
@@ -58,6 +58,9 @@ class BoardPageTabBarBuilderImpl implements DatabaseTabBarItemBuilder {
   ) =>
       const SizedBox.shrink();
 
+  @override
+  void dispose() {}
```

Comment:
is it okay to define the dispose function with an empty body in its superclass? so its subclass doesn't need to override it.